app/design/adminhtml/default/default/template/page/header.phtml

```diff
@@ -31,21 +31,37 @@
             <input id="global_search" name="query" type="text" class="input-text" value="<?php if(!empty($query)): ?><?php echo $query ?><?php else: ?><?php echo Mage::helper('core')->quoteEscape($defSearch) ?><?php endif ?>" onfocus="if(this.value=='<?php echo Mage::helper('core')->quoteEscape($defSearch, true) ?>')this.value=''; " onblur="if(this.value=='')this.value='<?php echo $defSearch ?>';" />
             <div id="global_search_autocomplete" class="autocomplete"></div>
             <script type="text/javascript">
-                new Ajax.Autocompleter(
-                    'global_search',
-                    'global_search_autocomplete',
-                    '<?php echo $this->getUrl('adminhtml/index/globalSearch') ?>',
-                    {
-                        paramName:"query",
-                        minChars:2,
-                        indicator:"global_search_indicator",
-                        updateElement:getSelectionId,
-                        evalJSON:'force'
+                document.getElementById('global_search').addEventListener('input', function() {
+                    let query = this.value;
+                    if (query.length >= 2) {
+                        document.getElementById('global_search_indicator').style.display = 'inline-block';
```

Comment:
should add this here:
			document.getElementById('global_search_autocomplete').style.display = 'block';

Reply:
done